src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx

```diff
@@ -290,7 +291,11 @@ Note that not all metrics have their custom tags as dimensions. Currently, only
 
 ## Metadata collection [#metadata-collection]
 
-Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch. This is an optional capability that's complementary to the CloudWatch Metric Streams integration.
+Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch.
+
+  <Callout variant="important">
+This optional capability is complementary to the CloudWatch Metric Streams integration. Your service discovery and monitoring will be incomplete if you don't enable this optional capability.
```

Comment:
do we want to mention any specific capabilities that won't work if Config is not enabled?

Reply:
I think that would be a bit too low level, and also we would need to keep an eye on it so it doesn't get updated. But if you think it is worth it, we can of course specify it